README.md

```diff
@@ -1,32 +1,48 @@
-Azkaban 3 [![Build Status](http://img.shields.io/travis/azkaban/azkaban.svg?style=flat)](https://travis-ci.org/azkaban/azkaban)
-========
+# Azkaban [![Build Status](http://img.shields.io/travis/azkaban/azkaban.svg?style=flat)](https://travis-ci.org/azkaban/azkaban)
 
 [![Join the chat at https://gitter.im/azkaban-workflow-engine/Lobby](https://badges.gitter.im/azkaban-workflow-engine/Lobby.svg)](https://gitter.im/azkaban-workflow-engine/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Building from Source
---------------------
+## Building Azkaban
+Azkaban builds use Gradle and requires Java 8 or higher.
 
-To build Azkaban packages from source, run:
+The following set of commands run on *nix platforms like Linux, OS X.
 
 ```
-./gradlew distTar
-```
+# Build Azkaban
+./gradlew build
 
-The above command builds all Azkaban packages and packages them into GZipped Tar archives. To build Zip archives, run:
+# Clean the build
+./gradlew clean
 
-```
-./gradlew distZip
-```
+# Build and install distributions
+./gradlew installDist
 
-If not building for the first time, it's good to clean first:
+# Run tests
+./gradlew test
 
+# Build without running tests
+./gradlew build -x test
 ```
-./gradlew clean
-```
 
-Documentation
--------------
+## Documentation
+Documentation is available at [azkaban.github.io](http://azkaban.github.io). 
+The source code for the documentation is in the [gh-pages](https://github.com/azkaban/azkaban/tree/gh-pages) branch.
+
+For help, please visit the [Azkaban Google Group](https://groups.google.com/forum/?fromgroups#!forum/azkaban-dev).
+
+## Developer Guide
+We recommend using [IntelliJ IDEA](https://www.jetbrains.com/idea/). Azkaban is a standard [Gradle](https://gradle.org/) 
+project. You can import it into your IDE using the `build.gradle` file in the root directory. For IntelliJ, choose Open 
+Project from the Quick Start box or choose Open from the File menu and select the root `build.gradle` file.
+
+### Code Style
+Azkaban follows Google code style. The template file, `intellij-java-google-style.xml`, can be found in the root 
+directory.
+
+#### IntelliJ IDEA 2017.1 on Mac OS X
```

Comment:
not at all. It's just convenient. I couldn't find a single liner page detailing just this part.